src/components/common/Price/index.tsx

```diff
@@ -63,24 +95,59 @@ export default function Price({ flowType, coinA, coinB }: Props) {
 			if (isMounted && p) {
 				setLaunchPadPrice(p);
 			}
-		}).catch((error: string) => console.log(error));
-
-		querySushiPoolPrices(sushiSwapPools[`${coinA}-${coinB}`])
-			.then(({ data }) => {
-				if (data?.error) {
-					console.error('fetching Sushi Pools price error: ', data.error);
-				} else {
-					const { pair } = data.data;
-					if (isMounted && pair) {
-						const { symbol: _coinA } = pair.token0;
-						const { symbol: _coinB } = pair.token1;
-
-						let realPrice = '';
-						if (_coinA.includes(coinA) && _coinB.includes(coinB)) {
-							realPrice = pair.token0Price;
-						} else if (_coinA.includes(coinB) && _coinB.includes(coinA)) {
-							realPrice = pair.token1Price;
+		});
+
+		querySushiPoolPrices(sushiSwapPools[`${getTokenSymbol(coinA)}-${getTokenSymbol(coinB)}`]).then(({ data }) => {
+			if (data?.error) {
+				console.error('fetching Sushi Pools price error: ', data.error);
+			} else {
+				const { pair } = data.data;
+				if (isMounted && pair) {
+					const { symbol: _coinA } = pair.token0;
+					const { symbol: _coinB } = pair.token1;
+
+					let realPrice = '';
+
+					if (_coinA.includes(getTokenSymbol(coinA)) && _coinB.includes(getTokenSymbol(coinB))) {
+						realPrice = pair.token0Price;
+					} else if (_coinA.includes(getTokenSymbol(coinB)) && _coinB.includes(getTokenSymbol(coinA))) {
+						realPrice = pair.token1Price;
+					}
+
+					if (
+						coinA.includes(Coin.IbAlluoBTC) ||
```

Comment:
Lots of if statements, nested as well. Is it possible to make thing into a switch statement, or at least use inverted if statements to avoid nesting